src/main/resources/assets/minestuck/models/block/cassette_player_closed.json

```diff
@@ -0,0 +1,243 @@
+{
+	"credit": "Made by Ishumire",
```

Comment:
I'm not confident that the resolution of this model fits into the vanilla setting. I'd suggest getting a simplified model to replace it for base minestuck